nixos/modules/services/networking/tailscale.nix

```diff
@@ -75,6 +81,13 @@ in {
       stopIfChanged = false;
     };
 
+    boot.kernel.sysctl = mkIf (cfg.useRoutingFeatures == "server" || cfg.useRoutingFeatures == "both") {
+      "net.ipv4.ip_forward" = true;
```

Comment:
Is this different to
```
      "net.ipv4.conf.all.forwarding" = true;
      "net.ipv6.conf.all.forwarding" = true;
```
?

Reply:
I took it from here: https://tailscale.com/kb/1019/subnets/#enable-ip-forwarding

From the [kernel documentation](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt):

```
ip_forward - BOOLEAN
	0 - disabled (default)
	not 0 - enabled

	Forward Packets between interfaces.

	This variable is special, its change resets all configuration
	parameters to their default state (RFC1122 for hosts, RFC1812
	for routers)
```

```
/proc/sys/net/ipv4/* Variables:
forwarding - BOOLEAN
	Enable IP forwarding on this interface.  This controls whether packets
	received _on_ this interface can be forwarded.
```

I think the main difference is whether it resets all configuration parameters to their default state. I'll test if `net.ipv4.conf.all.forwarding` works with Tailscale.

Reply:
After some quick testing, `net.ipv4.conf.all.forwarding` works instead of `net.ipv4.ip_forward` so I've changed it to use that :+1: